orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1184,6 +1190,11 @@ def run_step(model, input_left, input_right):
         _test_helpers.assert_gradients_match_and_reset_gradient(ort_model, pt_model, atol=1e-3, rtol=1e-3)
 
 def test_gradient_correctness_einsum_2():
+    # In PyTorch 1.11.0, there is issue during reduce node shape handling for exporter that
```

Comment:
shall we use `@pytest.mark.skipif`? And add a TODO so we remove it in the future.